### ISSUE TITLE
Sort columns

### ISSUE THREAD

**Issue by rrebase** [rrebase/knboard#14]
A column might have a lot of cards and thus sorting by created date, name will be quite useful. 

We already have an options menu (with delete column). Could just add the sort buttons there.

**Comment by matheusmosca** [comment]
Hi 👋, can I take this one? The cards have to be sorted before or after the API response?

**Comment by rrebase** [comment]
Yeah, you can take this one. The ordering should be done on the backend side as well, meaning that after a refresh the changed ordering remains.

I believe you should:
- Have a look at the API request that is made when a task is moved up/down in a column and do something similar, but for multiple tasks
